db/migrate/20180802145655_add_address_to_schools.rb

```diff
@@ -0,0 +1,7 @@
+class AddAddressToSchools < ActiveRecord::Migration
+  def change
+    add_column :schools, :street, :string
+    add_column :schools, :state, :string
```

Comment:
In Sierra, we're keeping track of: school_name, a couple of address lines (e.g. address_line_1, address_line_2, etc.), phone_number, and borough (dispersed throughout various fields).  I'd like to reflect that in MLN db.  Ping me on slack for help.